setup/stack/states/php7.2/php/init.sls

```diff
@@ -14,108 +14,49 @@ php_repo_key:
 
 php7.2:
   pkg.installed:
-    - refresh: true
     - pkgs:
       - libapache2-mod-php7.2
+      - php-json
```

Comment:
According to the purge commands in the other states, this should be php7.2-json instead of php-json
This also needs to be checked for the other states below (7.3 to 8.1).

Reply:
I add this for php7.2 - php7.4. Above this versions there is no php*.*-json anymore. So here comes php-json in place.